docs/Nachet_inference_documentation.md

```diff
@@ -0,0 +1,112 @@
+# Inference Request with n pipelines
+
+## Glossary
+
+### Pipelines
+Pipelines are define as a set of models that follow each others where output of
+one model is used as input for the next models. A pipeline contains from 1 to n
+models.
+
+```mermaid
+flowchart LR
+
+A{Pipeline entry point}-->|user input|SB1-->F{Pipeline exit point}-->|"last model
+output"|G>"return
+result.json to user"]
+
+subgraph SB1[In pipeline]
+    direction TB
+
+    B(model 01)-->|"model output
+    send to the next one"|C(model 02)-->|can have n models|D(n models)
+
+end
+```
+
+### Models
```

Comment:
A model is an AI model which is a part of a pipeline. A model accepts images as input
and returns JSON as output. Generally, this JSON contains the coordinates of the objects found in the
source image, that the model may pass along to feed the next step of the model.

Reply:
I think we should write : `next step of the pipeline` to not confuse model and pipeline.

Reply:
Sold!